.github/workflows/fuzzers.yml

```diff
@@ -28,82 +27,105 @@ jobs:
         sudo ./llvm.sh 9
 
     - uses: actions/checkout@v1
+
     - name: Create and prepare the initial seed corpus
       run: |
         fuzz/build_corpus.sh
         mv corpus.zip seed_corpus.zip
         mkdir seedcorpus
         unzip -q -d seedcorpus seed_corpus.zip
+
     - name: Download the corpus from the last run
       run: |
         wget --quiet https://dl.bintray.com/pauldreik/simdjson-fuzz-corpus/corpus/corpus.tar
         tar xf corpus.tar
         rm corpus.tar
+
     - name: List clang versions
       run: |
         ls /usr/bin/clang*
         which clang++
         clang++ --version
+
     - name: Build all the variants
       run: fuzz/build_fuzzer_variants.sh
+
     - name: Run the fast fuzzer (release build, default implementation, to explore fast)
       run: |
         set -eux
         for fuzzer in $allfuzzers; do
           mkdir -p out/$fuzzer # in case this is a new fuzzer, or corpus.tar is broken
-          build-fast/fuzz/fuzz_$fuzzer out/$fuzzer seedcorpus -max_total_time=30 $MAXLEN
+          # get input from everyone else (corpus cross pollination)
```

Comment:
What's the reasoning here? Assume I have a 6-year-old's understanding of fuzzing where some superintelligent AI somewhere generated a bunch of JSON and a bunch of JSON pointers to go with them, and then it runs each of them one by one. How does corpus cross pollination work?

Reply:
There is no magic here!
Most of the fuzzers use the same input (json files), but do different things with the parsed result. If one fuzzer has found interesting input, it will likely be useful to the other fuzzers. Corpus cross pollination is to try input from the other fuzzers and see if it benefits a particular fuzzer. This way, fuzzing effort is accumulated in case the input format is the same. When adding a new fuzzer to the mix, like this pull request does, it gets the benefit from the past effort of all the other fuzzers.